components/01-atoms/lists/list.stories.js

```diff
@@ -17,3 +17,11 @@ export const OrderedList = () => `
   ${listTwig({ list__items: listData.ordered__list__items, list__type: 'ol' })}
 </div>
 `;
+export const InlineList = () => `
+<div class="text-field">
+  ${listTwig({
+    list__items: listData.inline__list__items,
+    list__type: 'inline',
```

Comment:
@Gregg-Hendrix instead of a `list__type` of `inline`, this should be kept as a `ul` element. I'm not sure we need an `ol` version, but you could add an example of each.

Reply:
For this component, we may want to do something like this instead: 
```
export const InlineListUL = () => `
<div class="text-field">
  ${listTwig({
    list__items: listData.inline__list__items,
    list__type: 'ul',
    list__base_class: 'inline',
  })}
</div>
`;
export const InlineListOL = () => `
<div class="text-field">
  ${listTwig({
    list__items: listData.inline__list__items,
    list__type: 'ol',
    list__base_class: 'inline',
  })}
</div>
`;
```

The goal of an inline list is to offer a row of links, inline, similar to how breadcrumbs render but the use case would be for badges or tags. Essentially, something that matches this look: 

![Screenshot-20220810163333-775x571](https://user-images.githubusercontent.com/366413/184024361-c0671cf1-ded9-40f8-a925-30a81fcdb403.png)

Which, style-wise, could look like: 
```
.inline {
  list-style-type: none;
}

.inline__item {
  display: inline-block;
}
```

----
What you have here is a great start :clap: 

Does that make sense? I suppose we should have live examples listed on the Notion doc which would help clarify the expectation around what a component should look like. I'm going to give that some thought.

Reply:
@joetower Oh!! This does make a lot of sense! Thank you for sharing your wisdom ^_^! Some live examples in notion would definitely be helpful. Thank you for taking the time and sharing this feedback, lately I've been working a bit with `display: inline-block` thank you for showing me how that property can convert my list from "stacked" to `inline`.

When you get another chance, please take another look :)

Reply:
@Gregg-Hendrix Your changes looks good. Thank you!

I'm not sure these `inline` lists should actually have `list-style-type`s defined - `bullet` defined for the `ul` and `decimal` defined for the `ol` (note that in what I suggested above, neither `ol` or `ul` showed a list style inside the `inline` variation), but I'll hold off on further review until we discuss this in our L10. 

Looking forward to further discussion. :partying_face:

Reply:
@joetower Hey Joe!

I did remember the feedback in which you mentioned that having these list type defined isn't necessary. I just didn't want to "copy and paste"your answer! So adding these types was just a personal challenge for me. I know that we want to keep the styling to a minimum, and this kind of goes deeper into styling. I was just trying to prove to myself "can I do this"?

After it has been reviewed in the L10, I'll take off these additional styles :), then ask for a final (hopefully) review from you! Thank you so much!!

Reply:
@Gregg-Hendrix @joetower I went back to some other projects to see how I've had this set up before. I haven't seen any "inline list" components, but rather an "inline menu" -- which includes the `ul` list component, and then displays it inline (using menu-item length items in the `yml` file. 

I do, however, see a `dl` list (definition list.) Maybe this is something to consider, adding a definition list, and moving the inline list to be an inline menu (which is a more common usage, with links, I would think.)

Reply:
@mndonx 

Thank you SO MUCH for this feedback, it took me awhile to circle back to it, but please do take another look whenever you have time :)

Reply:
@Gregg-Hendrix A definition list should contain `<dt>` and `<dd>` markup instead of `<li>`. The final result of this markup should look like 
```
 <dl>
  <dt>Coffee</dt>
  <dd>Black hot drink</dd>
</dl> 
```
Note that a single `<dt>` and `<dd>` are required. See https://www.w3schools.com/tags/tag_dt.asp and https://www.w3schools.com/tags/tag_dd.asp for further details.

This PR should contain changes to `https://github.com/emulsify-ds/compound/blob/main/components/01-atoms/lists/_list-item.twig` so the `<li>` isn't being statically set but uses a `{{ list_item__type }}` variable that defaults to `<li>` but can be changed to `<dt>`/`<dd>` whenever a definition list is used.

Reply:
I originally edited this in the `yml` file, thank you for pointing me in the right direction!